app/scripts/metamask-controller.js

```diff
@@ -825,6 +832,11 @@ export default class MetamaskController extends EventEmitter {
         swapsController.setSwapsLiveness,
         swapsController,
       ),
+      // approval controller
+      resolvePendingApproval: approvalController.resolve.bind(
```

Comment:
It looks like these are never called :thinking:

Reply:
Ah, I had actions in the UI, but they were also yet to be called. I've removed these for now as well.